pkgs/development/node-packages/default.nix

```diff
@@ -436,13 +436,14 @@ let
       ];
     };
 
-    thelounge = super.thelounge.override {
+    thelounge = super.thelounge.overrideAttrs (_: {
       buildInputs = [ self.node-pre-gyp ];
       postInstall = ''
         echo /var/lib/thelounge > $out/lib/node_modules/thelounge/.thelounge_home
         patch -d $out/lib/node_modules/thelounge -p1 < ${./thelounge-packages-path.patch}
       '';
-    };
+      meta.maintainers = with lib.maintainers; [ winter ];
```

Comment:
If setting the maintainer does not work that way I think we should rather drop this commit.